webhooks-extension/config/triggers_prototype/github/README.md

```diff
@@ -0,0 +1,18 @@
+# GitHub Webhooks
+
+You can create or delete GitHub webhooks using the provided Task and TaskRun.
+
+- Apply the Task definition:
+`kubectl apply -f config/triggers_prototype/github/webhook.yaml`
+
+- Create a GitHub secret, for example with the Tekton Dashboard or importing it using the Tekton Webhooks Extension. You will refer to its name in the next step.
```

Comment:
I dont think we should advocate importing secrets from a git repo.  Also I would saying "using the Tekton Dashboard" rather than "with"